client/src/components/ActiveChat/OtherUserBubble.js

```diff
@@ -1,48 +1,72 @@
 import React from "react";
 import { makeStyles } from "@material-ui/core/styles";
 import { Box, Typography, Avatar } from "@material-ui/core";
+import { setReadMessage } from "../../store/utils/thunkCreators";
+import { useDispatch } from "react-redux";
 
 const useStyles = makeStyles(() => ({
   root: {
-    display: "flex"
+    display: "flex",
   },
   avatar: {
     height: 30,
     width: 30,
     marginRight: 11,
-    marginTop: 6
+    marginTop: 6,
   },
   usernameDate: {
     fontSize: 11,
     color: "#BECCE2",
     fontWeight: "bold",
-    marginBottom: 5
+    marginBottom: 5,
   },
   bubble: {
     backgroundImage: "linear-gradient(225deg, #6CC1FF 0%, #3A8DFF 100%)",
-    borderRadius: "0 10px 10px 10px"
+    borderRadius: "0 10px 10px 10px",
   },
   text: {
     fontSize: 14,
     fontWeight: "bold",
     color: "#FFFFFF",
     letterSpacing: -0.2,
-    padding: 8
-  }
+    padding: 8,
+  },
 }));
 
 const OtherUserBubble = (props) => {
   const classes = useStyles();
-  const { text, time, otherUser } = props;
+  const { text, time, otherUser, id, recipientId } = props;
+  const dispatch = useDispatch();
+
+  async function handleSubmit(msgId) {
+    const body = {
+      msgId: msgId,
+      senderId: otherUser.id,
+      recipientId: recipientId,
+      read: true,
+    };
+    dispatch(setReadMessage(body));
+  }
+
   return (
     <Box className={classes.root}>
-      <Avatar alt={otherUser.username} src={otherUser.photoUrl} className={classes.avatar}></Avatar>
+      <Avatar
+        alt={otherUser.username}
+        src={otherUser.photoUrl}
+        className={classes.avatar}
+      ></Avatar>
       <Box>
-        <Typography className={classes.usernameDate}>
-          {otherUser.username} {time}
-        </Typography>
+        {props.read ? (
+          <Typography className={classes.usernameDate}>
+            {otherUser.username} {time} read
+          </Typography>
+        ) : (
+          <Typography className={classes.usernameDate}>
+            {otherUser.username} {time} unread
+          </Typography>
+        )}
```

Comment:
Since this ternary operator only affects the read/unread text, you can just use it for the text rather than the whole component.